xray/src/bin/merge_xray_quadtrees.rs

```diff
@@ -0,0 +1,18 @@
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "merge_xray_quadtrees")]
+/// Merge partial xray octrees. We assume that the root
+/// of each quadtree belongs to the same level of the final
+/// quadtree.
+struct CommandlineArguments {
+    /// Directory with partial xray quadtrees.
+    #[structopt(parse(from_os_str))]
+    directory: PathBuf,
```

Comment:
How about a Vec<PathBuf>, so we could potentially have inputs in several places?
And how about also specifying one dedicated output directory (which can be equal to one of the input directories)?

Reply:
Agree with `Vec<PathBuf>` suggestion.
About output directory: what would be output there? Only the newly generated nodes/images or the whole quadtree?

Reply:
I think it should be the whole quadtree